src/webrtc_session.ts

```diff
@@ -461,8 +461,8 @@ export class HTTPWebRTCSession extends WebRTCSession {
     }
     onIceCandidate(ev: RTCPeerConnectionIceEvent) {
         console.log("got ice candidate", ev)
-        if (ev.candidate != null)
-            return
+        /* if (ev.candidate != null)
```

Comment:
null means we're finished gathering ICE candidates. In HTTP based connection we are not doing "Trickle ICE" - where every candidate is sent as it's received - but wait for the entire ICE gathering to finish. Without this if, T7 send a POST request for every candidate it gets and that's not what our server expects.